packages/nocodb-sdk/src/lib/errorUtils.ts

```
@@ -0,0 +1,7 @@
+export class NcSDKError extends Error {
+  constructor(message: string) {
+    super(message);
+  }
```

Biome

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)